### PR TITLE
{lang}[gfbf/2023b] R v4.3.3

### PR DESCRIPTION
**Pull request by lazariv**
(created using `eb --new-pr`)

Split #19965 to separate PRs

Updated to latest version R/4.3.3 as https://github.com/easybuilders/easybuild-easyconfigs/pull/19965#discussion_r1501401393

Updated R packages with easy_update tool as suggested https://github.com/easybuilders/easybuild-easyconfigs/pull/19965#discussion_r1501401751

- [x] Requires https://github.com/easybuilders/easybuild-easyconfigs/pull/20046

### CODE REVIEW

File: easybuild/easyconfigs/r/R/R-4.3.3-gfbf-2023b.eb [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+    ('LibTIFF', '4.6.0'),
+    ('Java', '11', '', SYSTEM),
+    ('libgit2', '1.7.2'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+    ('cURL', '8.3.0'),
+    ('Tk', '8.6.13'),  # for tcltk
+    ('HarfBuzz', '8.2.2'),  # for textshaping
+    ('FriBidi', '1.0.13'),  # for textshaping
+
+]
```
Comment (Flamefire) [inline easybuild/easyconfigs/r/R/R-4.3.3-gfbf-2023b.eb:41]: Empty line:
```suggestion
]
```

**Comment by lazariv** [comment]
Test report by @lazariv
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
i8022 - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7352 24-Core Processor, 1 x NVIDIA NVIDIA A100-SXM4-40GB, 545.23.08, Python 3.11.5
See https://gist.github.com/lazariv/7cf6e4512fee7198a50f201bac5e65f4 for a full test report.

**Comment by lazariv** [comment]
Test report by @lazariv
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
i8022 - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7352 24-Core Processor, 1 x NVIDIA NVIDIA A100-SXM4-40GB, 545.23.08, Python 3.11.5
See https://gist.github.com/lazariv/bdb048a67bf837cc93df987eab19c620 for a full test report.

**Comment by lazariv** [comment]
Test report by @lazariv
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
i8022 - Linux Rocky Linux 8.7 (Green Obsidian), x86_64, AMD EPYC 7352 24-Core Processor, 1 x NVIDIA NVIDIA A100-SXM4-40GB, 545.23.08, Python 3.8.13
See https://gist.github.com/lazariv/d055472924bab1f04f688cfeb5b8517f for a full test report.

**Review by Flamefire — APPROVED** [review]
- [x] Requires #20046

**Comment by lazariv** [comment]
Test report by @lazariv
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
i8022 - Linux Rocky Linux 8.7 (Green Obsidian), x86_64, AMD EPYC 7352 24-Core Processor, 4 x NVIDIA NVIDIA A100-SXM4-40GB, 545.23.08, Python 3.8.13
See https://gist.github.com/lazariv/8a8310560732a4d711919447f37ef0cf for a full test report.

**Comment by lazariv** [comment]
Test report by @lazariv
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
i8022 - Linux Rocky Linux 8.7 (Green Obsidian), x86_64, AMD EPYC 7352 24-Core Processor, 4 x NVIDIA NVIDIA A100-SXM4-40GB, 545.23.08, Python 3.8.13
See https://gist.github.com/lazariv/86b2a4ad769dbc599d8c0d62d68ecbd5 for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
n1296 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) Platinum 8470 (icelake), Python 3.8.13
See https://gist.github.com/Flamefire/7253cd9c594d64f0d15813478ceb6c53 for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
i8022 - Linux Rocky Linux 8.7 (Green Obsidian), x86_64, AMD EPYC 7352 24-Core Processor (zen2), 8 x NVIDIA NVIDIA A100-SXM4-40GB, 545.23.08, Python 3.8.13
See https://gist.github.com/Flamefire/870008bf8f73961334bcbc2bdc82a63d for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
i8022 - Linux Rocky Linux 8.7 (Green Obsidian), x86_64, AMD EPYC 7352 24-Core Processor (zen2), 8 x NVIDIA NVIDIA A100-SXM4-40GB, 545.23.08, Python 3.8.13
See https://gist.github.com/Flamefire/3dda06dea9750b08d12b4b3003e46abd for a full test report.

**Comment by lazariv** [comment]
Test report by @lazariv
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
n1590 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) Platinum 8470, Python 3.8.13
See https://gist.github.com/lazariv/d4b2f6040cc343847d3c440a7ac8deb1 for a full test report.

**Comment by lazariv** [comment]
Test report by @lazariv
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
n1590 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) Platinum 8470, Python 3.8.13
See https://gist.github.com/lazariv/77c90225da5dc13f0729b75ab66b0bb8 for a full test report.

**Comment by lazariv** [comment]
Test report by @lazariv
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
i8036 - Linux Rocky Linux 8.7 (Green Obsidian), x86_64, AMD EPYC 7352 24-Core Processor, 1 x NVIDIA NVIDIA A100-SXM4-40GB, 545.23.08, Python 3.8.13
See https://gist.github.com/lazariv/f4149e4b02c87ce13217444eed6c9e7e for a full test report.

File: easybuild/easyconfigs/r/R/R-4.3.3-gfbf-2023b.eb [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+# !! order of packages is important !!
+# packages updated on 4th March 2024
+exts_list = [
```
Comment (Micket) [inline easybuild/easyconfigs/r/R/R-4.3.3-gfbf-2023b.eb:66]:
```suggestion
exts_list = [
    # include packages that are part of the base installation of R,
    # both to make sure they are available (via sanity check),
    # and to be able to pass the check for required dependencies when installing extensions in parallel
```

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 10 out of 10 (1 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/73f3eac544a42150c006cdca0eb236e7 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @lazariv!

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3
CORE_CNT=16

**Pull request merged**